argo-cd-apps/base/all-clusters/infra-deployments/crossplane-control-plane/crossplane-control-plane.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
According to the Jira task, this is meant for member clusters only, please move this to member folder